### PR TITLE
Import scripts for wp

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé

Create WP posts out of a .json file from a GC Notify template
[Issue - [Validate that Government Content works in GC Articles](https://github.com/cds-snc/GC-Articles/issues/8)]

# Test instructions | Instructions pour tester la modification
```bash
npm run import --file=/path/to/file.json
```
Screen Shot 2021-08-24 at 9 04 09 AM

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet
- I was thinking about writing tests using jest but skipped it for now
- What do we do about the french version

### CODE REVIEW

File: import/importer.mjs [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { readFileSync } from 'fs';
+import { spawn } from 'child_process';
+import markdownConverter from "./markdown-converter.mjs";
+import { exit } from 'process';
```
Comment (timarney) [inline import/importer.mjs:4]: Looks like `exit` and `create` are not in use

File: import/importer.mjs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+export default class Importer {
+    static processNotifyTemplate = (filename) => {
+        let rawdata = readFileSync(filename);
+        let contents = JSON.parse(rawdata);
+    
+        let i = 0
+        contents.forEach(item => {
+            
+            const convertedBody = markdownConverter(item.content)
```
Comment (timarney) [inline import/importer.mjs:15]: Looks like `const convertedBody = markdownConverter(item.content)` is not in use

File: import/importer.mjs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import markdownConverter from "./markdown-converter.mjs";
+
+export default class Importer {
+    static processNotifyTemplate = (filename) => {
+        let rawdata = readFileSync(filename);
+        let contents = JSON.parse(rawdata);
+    
+        contents.forEach(item => {
+            const post = {
+                title: markdownConverter(item.subject),
```
Comment (timarney) [inline import/importer.mjs:12]: Lets skip converting markdown for the title

Comment (timarney) [reply import/importer.mjs:12]: Given the contents of the file:

Given the subject lines are all the same looks like ideally we would parse the title out of the content

I think they all follow the same pattern :

```# A non-medical mask can help protect those around you\r\n\r\n```


We could split on the `\r\n\r\n` and remove the # 

Thoughts?

Comment (daine) [reply import/importer.mjs]: Ended up parsing the body and get the first paragraph of the content as the title (or if it's too lengthy, use the subject)

**Review by timarney — APPROVED** [review]

**Pull request merged**